src/package_json.rs

```diff
@@ -0,0 +1,17 @@
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize, Default)]
+struct EnginesField {
+    node: Option<node_semver::Range>,
+}
+
+#[derive(Debug, Deserialize, Default)]
+pub struct PackageJson {
+    engines: Option<EnginesField>,
+}
+
+impl PackageJson {
+    pub fn node_range(&self) -> Option<node_semver::Range> {
+        self.engines.as_ref()?.node.clone()
+    }
```

Comment:
maybe `Option<&node_semver::Range>` and let the consumer decide on whether to clone? <picture data-single-emoji=":eyebrow-man:" title=":eyebrow-man:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..ti2JjK9q9hJlJUzP.jlBCAzCwxC9FiDFp2ZpKYjFjKXdLEbTBpzD3FRMn9YZfM1Iyi8O1dgyB2uiUbkMJmCu13nMqALidajYt2MXD0ilqvhdPNOJTVDXclR0CpvbT3GagowDODsb3tYmapw.TpyWe1z95rQChzZyk8BOzA" alt=":eyebrow-man:" align="absmiddle"></picture>

Reply:
Trying doing this here:
https://github.com/amitdahan/fnm/pull/3/commits/305fae3699ed3fc10b19145af615da7063e6a17d

Not sure it's what you meant :sadpepe: